packages/docs/pages/reference/elements/ellipsis.mdx

```diff
@@ -7,7 +7,7 @@ import {
 
 # `<ellipsis>`
 
-Short description of what the `ellipsis` element does.
+Inserts the ellipsis (...). Used as the self-closing tag `<ellipsis/>`.
```

Comment:
Can you use the unicode version of the ellipsis here? and put a space before the closing of the self-closing tag? I.e. `<ellipsis />`